docs/user-guide/test-zen-doc_Sean.md

```diff
@@ -53,50 +52,72 @@ FTP Port  |The FTP Port number for internal use. The default port is 21. If not
+Perform Zowe server configuration in the wizard by providing inputs to the prompts for configuration values.
+Some steps may require an administrator with sufficient privileges to complete the step. You have the option to **Skip** the page to allow the step to be completed externally to the wizard. It is also possible to input the correct values on the page, and click **View Job** to collect the job content to provide to an administrator without submitting it.
+Each value entered is validated.
+
+When you complete an entry the following actions can be performed:
+
+* **View/Edit YAML**  
+This feature lets you preview or adjust the YAML configuration that is used by Zowe. The prompts of the wizard are used to automatically generate the YAML contents, but you also have the options to review, edit, or import and export contents of the YAML file.
+* **View/Submit Job**  
+When you have completed providing the values for each step, you can click **View/Submit Job** to execute the operations of the configuration step upon the z/OS system. After a job is submitted, job output is available for review.
+
+* **Skip a step**  
+If you need to skip a step that you cannot perform such as an administrative seurity action that you cannot perform yourself, we recommend you click **View/Edit YAML**. This option allows you to copy the YAML in its current state. You can then provide the YAML and relevent JCL sample, and explanation for the particular step from this table to an administrator that is authorized to perform the task.
+
+|Install Wizard Initalization Step|Description|Sample JCL|
+|---|---|---|
+|Installation|**Purpose:**<br /> Create datasets for Zowe's PARMLIB content and non-ZFS extension content for a given Zowe Instance<br /><br />**Action:**<br />1.  Allocate the PDSE FB80 dataset with at least 15 tracks named from Zowe parameter `zowe.setup.dataset.parmlib`<br />2. Allocate the PDSE FB80 dataset with at least 30 tracks named from Zowe parameter `zowe.setup.dataset.authPluginLib`<br />3. Copy the ZWESIP00 member from `zowe.setup.dataset.prefix`.SZWESAMP into `zowe.setup.dataset.parmlib`|[ZWEIMVS](https://github.com/zowe/zowe-install-packaging/tree/feature/v3/jcl/files/SZWESAMP/ZWEIMVS)|
+|APF Auth|**Purpose:**<br /> Zowe contains one privileged component, ZIS, which enables the security model. The majority of Zowe is unprivileged and in key 8. The load library for the ZIS component and its extension library must be set as APF authorized and run in key 4. This enables ZIS and components that depend on ZIS.<br /><br />**Action:**<br />1. APF authorize the datasets defined at `zowe.setup.dataset.authLoadlib` and `zowe.setup.dataset.authPluginLib`.<br />2. Define PPT entries for the members ZWESIS01 and ZWESAUX as Key 4, NOSWAP in the SCHEDxx member of the system PARMLIB.|[ZWEIAPF](https://github.com/zowe/zowe-install-packaging/tree/feature/v3/jcl/files/SZWESAMP/ZWEIAPF)|
```

Comment:
"2. Define PPT entries for the members ZWESIS01 " I believe this is not happening with APF auth and must be done  separately

Reply:
Good catch
@1000TurquoisePogs I believe he is right here?
Let's make a distinction that it's good to do, but it's not what Zen does, and more info can be found here:
https://docs.zowe.org/stable/user-guide/configure-xmem-server/#key-4-non-swappable

@janan07